staging/src/k8s.io/legacy-cloud-providers/azure/azure_storageaccount.go

```diff
@@ -90,7 +96,12 @@ func (az *Cloud) GetStorageAccesskey(account, resourceGroup string) (string, err
 }
 
 // EnsureStorageAccount search storage account, create one storage account(with genAccountNamePrefix) if not found, return accountName, accountKey
-func (az *Cloud) EnsureStorageAccount(accountName, accountType, accountKind, resourceGroup, location, genAccountNamePrefix string, enableHTTPSTrafficOnly bool) (string, string, error) {
+func (az *Cloud) EnsureStorageAccount(account StorageAccount, genAccountNamePrefix string, enableHTTPSTrafficOnly bool) (string, string, error) {
```

Comment:
should pass a pointer of `AccountOptions` in the function

Reply:
done